### PR TITLE
Feature/custom headings in trysilplug

### PR DESCRIPTION
**Pull request by misund**

### CODE REVIEW

File: src/atoms/DymoHeading.js [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
 
 import { Heading } from '..';
 
 const FormattedHeading = styled(Heading)`
   display: inline-block;
   margin: 0 0 calc(2/3*${props => props.theme.variables.verticalBase});
   ${props => (props.hasImage ? 'max-width: 90%;' : `max-width: calc(100% - ${props.theme.variables.horizontalBase});`)}
-
-	@media (min-width: ${props => props.theme.flexboxgrid.breakpoints.sm}em) {
-		${props => ((props.flexingSize === 'full') ? css`
```
Comment (SindreSvendby) [inline src/atoms/DymoHeading.js]: flexingSize is not used anymore. Should remove from props and default value

File: src/molecules/TrysilPlug.js [hunk 2]
```diff
@@ -64,7 +65,14 @@ TrysilPlug.propTypes = {
 	offset: PropTypes.number,
 	sources: PropTypes.arrayOf(PropTypes.object),
 	ratio: PropTypes.number.isRequired,
-	ALLCAPS: PropTypes.bool,
+	Heading: PropTypes.element,
```
Comment (SindreSvendby) [inline src/molecules/TrysilPlug.js]: If we are sending in and header. Should we then send inn props or just add the props to the heading before sending inn? I see we have a default value here. But should we? Or should we have a component that has heading as mandatory? And then two components that uses trysilplug but with different headings? Im not sure. Just wanted to throw the idea out here

Comment (misund) [reply src/molecules/TrysilPlug.js]: If I trow in the default, it's backwards compatible.

Comment (misund) [reply src/molecules/TrysilPlug.js]: I'm open to adding the props before we send the component in.

Comment (misund) [reply src/molecules/TrysilPlug.js]: Removed the prop from propTypes and defaultProps

File: src/atoms/DymoHeading.js (hunk 1 above)
Comment (misund) [reply src/atoms/DymoHeading.js]: Done, good catch

File: src/molecules/TrysilPlug.js (hunk 2 above)
Comment (SindreSvendby) [reply src/molecules/TrysilPlug.js]: If we keep the default then we need to support sending header props in anyways. So then its no use. Lets keep it as it is.

**Pull request merged**